docs/how-to/rockcraft-pack-action.rst

```diff
@@ -0,0 +1,23 @@
+How to build a ROCK with Rockcraft's GitHub Action
+**************************************************
+
+Within your GitHub repository, make sure you have
+`GitHub Actions enabled <https://docs.github.com/en/actions/quickstart>`_.
+
+Navigate to ``.github/workflows``, open the YAML file where you want the ROCK
+build to take place, and add the following steps:
+
+.. literalinclude:: code/rockcraft-pack-action/rockcraft-pack.yaml
+    :language: yaml
+
+Commit the changes and push. GitHub shall then trigger a new workflow
+execution and with it, Rockcraft will be set up and used to pack your ROCK,
+based on whatever ``rockcraft.yaml`` file there is at your project's root.
```

Comment:
This sounds very formal. My read of the style guide infers informality ("friendly, approachable and straight-talking") while still being precise, so maybe something like this:

```suggestion
Commit and push the changes. This will trigger a new workflow run using 
Rockcraft to pack your ROCK based on the ``rockcraft.yaml`` file at your 
project's root.
```